.ci/Jenkinsfile

```diff
@@ -34,13 +34,14 @@ pipeline {
         deleteDir()
         gitCheckout(basedir: "${BASE_DIR}")
         stash allowEmpty: true, name: 'source', useDefaultExcludes: false
+        setEnvVar("GO_VERSION", readFile("${env.WORKSPACE}/${env.BASE_DIR}/.go-version").trim())
       }
     }
     stage('Check') {
       options { skipDefaultCheckout() }
       steps {
         cleanup()
-        withGoEnv(){
+        withGoEnv(version: "${GO_VERSION}"){
           dir("${BASE_DIR}"){
             sh(label: 'check',script: 'make check')
           }
```

Comment:
```suggestion
          dir("${BASE_DIR}"){
            withGoEnv(version: "${GO_VERSION}"){
              sh(label: 'check',script: 'make check')
            }
```

Reply:
fixed in https://github.com/elastic/fleet-server/pull/346/commits/21328bf142b9578aa8ca73b4c912c0259cf03aa1